scheduler/src/cook/mesos/api.clj

```diff
@@ -31,6 +32,7 @@
             [cook.mesos.unscheduled :as unscheduled]
             [cook.mesos.util :as util]
             [cook.mesos]
+            [cook.util]
```

Comment:
Nitpick: can we alias this to something, e.g. `cutil`

Reply:
I'd prefer not to alias this since we have both `cook.mesos.util` and `cook.util` in use here.